__tests__/__packages__/cli-test-utils/src/TestUtils.ts

```diff
@@ -135,6 +135,7 @@ export function mockHandlerParameters(params: PartialHandlerParameters): IHandle
             ...params.arguments || {}
         },
         positionals: params.positionals || [],
+        // eslint-disable-next-line deprecation/deprecation
         profiles: params.profiles || new CommandProfiles(new Map()),
```

Comment:
Curious if we should remove this reference to `IHandlerParameters.profiles` from V3? I don't want to risk breaking existing tests though 😋

Reply:
I can take a look. I don't know if anything is still using it.